tests/manage/mcg/test_object_expiration.py

```diff
@@ -0,0 +1,66 @@
+import logging
+import uuid
+from time import sleep
+
+import pytest
+
+from ocs_ci.framework.pytest_customization.marks import tier1, bugzilla
+from ocs_ci.framework.testlib import MCGTest
+from ocs_ci.framework.testlib import skipif_ocs_version
+from ocs_ci.ocs.bucket_utils import s3_put_object, s3_get_object
+
+logger = logging.getLogger(__name__)
+
+
+class TestObjectExpiration(MCGTest):
+    """
+    Tests suite for object expiration
+
+    """
+
+    @skipif_ocs_version("<4.10")
+    @bugzilla("2034661")
+    @pytest.mark.polarion_id("OCS-3929")
+    @tier1
+    def test_object_expiration(self, mcg_obj, bucket_factory):
+        """
+        Test object is not deleted in minutes when object is set to expire in a day
+
+        """
+        bucket = bucket_factory(interface="OC")[0].name
```

Comment:
```suggestion
        bucket_name = bucket_factory()[0].name
```
I believe that using an S3 bucket here will make the creation part run a bit faster

Reply:
ack